### PR TITLE
Added aapt options to allow dirs beginning with underscores in assets

### PR DESCRIPTION
**Pull request by carlpoole**

### CODE REVIEW

File: assets/plugin-template/android/build.gradle [hunk 1]
```diff
@@ -24,6 +24,9 @@ android {
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        aaptOptions {
+            ignoreAssetsPattern '!.svn:!.git:!.ds_store:!*.scc:!CVS:!thumbs.db:!picasa.ini:!*~'
```
Comment (dwieeb) [inline assets/plugin-template/android/build.gradle:28]: We can probably exclude svn and cvs. Git is the one true way. 

No idea what scc files are.

Picasa is long dead.

**Comment by jcesarmobile** [comment]
I think this should go in the app template, not in the plugin template 
https://github.com/ionic-team/capacitor/blob/main/android-template/app/build.gradle

**Comment by carlpoole** [comment]
Julio's right. Moving to the app template.

**Comment by carlpoole** [comment]
Moved to: https://github.com/ionic-team/capacitor/pull/3496

**Pull request closed without merging**